src/geocat/f2py/linint2_wrapper.py

```diff
@@ -27,6 +27,110 @@ def _linint1(xi, fi, xo, icycx, msg_py, shape):
+            
+            For geo-referenced data, yi is generally the latitude array.
+        
+        fi (:class:`numpy.ndarray`):
+            An array of two or more dimensions. If xi is passed in as an
+            argument, then the size of the rightmost dimension of fi
+            must match the rightmost dimension of xi. Similarly, if yi
+            is passed in as an argument, then the size of the second-
+            rightmost dimension of fi must match the rightmost dimension
+            of yi.
+            
+            If missing values are present, then linint2 will perform the
+            bilinear interpolation at all points possible, but will
+            return missing values at coordinates which could not be
+            used.
+        
+        xo (:class:`numpy.ndarray`):
+            A one-dimensional array that specifies the X coordinates of
+            the return array. It must be strictly monotonically
+            increasing, but may be unequally spaced.
+            
+            For geo-referenced data, xo is generally the longitude
+            array.
+            
+            If the output coordinates (xo) are outside those of the
+            input coordinates (xi), then the fo values at those
+            coordinates will be set to missing (i.e. no extrapolation is
+            performed).
+       
+        yo (:class:`numpy.ndarray`):
+            A one-dimensional array that specifies the Y coordinates of
+            the return array. It must be strictly monotonically
+            increasing, but may be unequally spaced.
+            
+            For geo-referenced data, yo is generally the latitude array.
+            
+            If the output coordinates (yo) are outside those of the
+            input coordinates (yi), then the fo values at those
+            coordinates will be set to missing (i.e. no extrapolation is
+            performed).
+        
+        icycx (:obj:`bool`):
+            An option to indicate whether the rightmost dimension of fi
+            is cyclic. This should be set to True only if you have
+            global data, but your longitude values don't quite wrap all
+            the way around the globe. For example, if your longitude
+            values go from, say, -179.75 to 179.75, or 0.5 to 359.5,
+            then you would set this to True.
+        
+        msg_py (:obj:`numpy.number`):
+            A numpy scalar value that represent a missing value in fi.
+            This argument allows a user to use a missing value scheme
+            other than NaN or masked arrays, similar to what NCL allows.
+    
+    Returns:
+        :class:`numpy.ndarray`: The interpolated grid. The returned
+        value will have the same dimensions as fi, except for the
+        rightmost two dimensions which will have the same dimension
+        sizes as the lengths of yo and xo. The return type will be
+        double if fi is double, and float otherwise.
+    '''
```

Comment:
Help text is not needed in the inner wrapper. This appears to just duplicate the text in the outer wrapper.